src/__tests__/task.test.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import { deleteTodo } from '../task';
+import { getTodoList, saveTodoList } from '../functions';
+
+describe('add and delete tasks', () => {
+  test('adds a single to-do item to the list', () => {
+    const todo = [];
+    saveTodoList(todo); // save the empty array of to-do list items
+    const numberOfItemsBeforeSave = getTodoList().length;
```

Comment:
- [ ] good job on wrapping the tests inside the describe function 👍 According to the requirements however you should create mocks for functions that use `localStorage` for example.

Reply:
Sure, we implemented mocks!